infrastructure/validators/onboarding/run-validator/using-systemd.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Use Systemd
+description: Using a service manager for your validator node.
+---
+
+You can run your validator as a [systemd](https://en.wikipedia.org/wiki/Systemd){target=\_blank} process so that it will automatically restart on server reboots or crashes (and helps to avoid getting [slashed]()).
```

Comment:
```suggestion
You can run your validator as a [systemd](https://en.wikipedia.org/wiki/Systemd){target=\_blank} process so that it will automatically restart on server reboots or crashes (and helps to avoid getting [slashed](TODO:update-path){target=\_blank}).
```